spec/abci/abci++_app_requirements.md

```diff
@@ -837,6 +846,12 @@ First, some simple start conditions:
 
 If `appBlockHeight == 0`, then call InitChain.
 
+Note that, in case there is a failure to commit the first block and
+a crash occurs,
+upon restart, `InitChain` will be called again. As explained,
+the application should not see this as an issue,  
+as it is expected to crash together with Comet. 
```

Comment:
To call it Comet reveals a certain intimacy with him. :P

But in the remaining of the document, CometBFT has been adopted.